packages/expo-contacts/android/src/main/java/expo/modules/contacts/ContactsModule.java

```diff
@@ -153,6 +153,9 @@ public void run() {
           } else {
             output.putParcelableArray("data", new Parcelable[0]);
           }
+          output.putBoolean("hasNextPage", false);
+          output.putBoolean("hasPreviousPage", false);
+          output.putInt("total", 1);
```

Comment:
looking at the block below this where `hasNextPage` etc are set for contact queries on name, it looks like we don't add a `total` there either. given that [this property is deprecated and the suggestion is to use `data.length` instead](https://docs.expo.io/versions/latest/sdk/contacts/#contactresponse), maybe it would be better for us to now remove the `total` property entirely? would be good for a follow up pr anyhow